CHANGELOG.md

```diff
@@ -35,6 +35,7 @@
 
 ### Improvements
 
+- Change API optional structs to pointers to conform with k8s guide ([#1170](https://github.com/kedacore/keda/issues/1170))
```

Comment:
Could you please move this to `###Other` section below, as this is not something that is directly affecting users.

Reply:
Sure thing, done :)